### ISSUE TITLE
Typescript error with example code

### ISSUE THREAD

**Issue by lludol** [zeyneloz/onesignal-node#46]
Hello, 

I installed the latest version, I copy / paste the example code and there is a Typescript error.

The example code : 
```javascript
// ...
// using async/await
try {
  const response = await client.createNotification(notification);
  console.log(response.body.id); // <-- HERE
} catch (e) {
  if (e instanceof OneSignal.HTTPError) {
    // When status code of HTTP response is not 2xx, HTTPError is thrown.
    console.log(e.statusCode);
    console.log(e.body);
  }
```

The Typescript error:
```
Property 'id' does not exist on type '{}'
```

Why there is this error? Because of this in index.d.ts:
```javascript
export interface ClientResponse {
    statusCode: number;
    body: {};
    headers: {
        [key: string]: any;
    };
}
```

**Comment by zeyneloz** [comment]
Hey @lludol ,

This is just a sample code and not meant to execute correctly for every situation. The exception  basically says that there is no `id` property on `body`. You might try to log body itself and see the content: `console.log(response.body);`

**Comment by lludol** [comment]
Yes I know that can check what is inside. But what's the point of having Typescript types defininitions if there are incomplete?

**Comment by zeyneloz** [comment]
My bad, I overlooked the error details you provided. I will open a bug-fix request for this ticket. Thanks for opening an issue.

**Comment by zeyneloz** [comment]
Fixed on https://github.com/zeyneloz/onesignal-node/pull/47
Released as v3.0.2
